e2e/tests/api_unspec_claim_tests/api_1v1_adjourn_vacate_ga_test.js

```diff
@@ -38,6 +38,7 @@ Scenario('AC 2 - 14 Days without consent, without notice', async ({api}) => {
   let hearingDate = await api.createDateString(14);
   civilCaseReference = await api.createUnspecifiedClaim(
     config.applicantSolicitorUser, mpScenario, 'Company');
+  await api.amendClaimDocuments(config.applicantSolicitorUser);
```

Comment:
@pliao-hmcts could you please remove the @ignore-api-test and kick the build again? it is in line no. 7

Reply:
the test was not ignored in test branch PR and the pipeline is green
https://github.com/hmcts/civil-general-apps-ccd-definition/pull/742/files#diff-7712f21f5b6396a0fb45c01556de74742a6dabdf007abc299c088eb2cb369b99